api.raml

```diff
@@ -23,28 +22,79 @@ securitySchemes:
           type: string
 
 
-#
-# Example Endpoints
-#
+/sequenceForGff3/{sequenceType}:
+  displayName: Retrieve sequence for GFF3
+  post:
+    securedBy: header-auth
+    queryParameters:
+      deflineFormat: lib.DeflineFormat
+      forceStrandedness:
+        type: boolean
+        default: false
+      basesPerLine:
+        type: integer
+        default: 60
+    body:
+      multipart/form-data:
+        type: object
+        properties:
+            file: file
+    responses:
+      200:
+        body:
+          application/octet-stream:
+      401:
+        body:
+          application/json:
+            type: error.UnauthorizedError
+      500:
+        body:
+          application/json:
+            type: error.ServerError
 
-/hello:
-  displayName: Hello World
-  get:
+/sequenceForBed/{sequenceType}:
+  displayName: Retrieve sequence for BED
+  post:
+    securedBy: header-auth
+    queryParameters:
```

Comment:
I feel like these should go into the body of the request as additional form parameters, esp if the value of deflineFormat is complex enough to warrant a separate type.  Typically query params are used only for refining a request, not part of the declaration

Reply:
I had it all structured together - JSON containing a file under "sequence" or something like that - but I couldn't work out how to post to the service using `curl`, so I tried this version where a file is a whole body, which made me move everything else to query parameters. But actually I had no success with this version too - something in Jersey gets in the way of reading the file.